Formula/rabbitmq.rb

```diff
@@ -105,7 +105,9 @@ def plist
   test do
     ENV["RABBITMQ_MNESIA_BASE"] = testpath/"var/lib/rabbitmq/mnesia"
     system sbin/"rabbitmq-server", "-detached"
-    sleep 5
+
+    system "sleep", "30"
```

Comment:
```suggestion
    sleep 30
```

Ideally this would be a lower sleep that loops until e.g. a port is open?